config/initializers/route_config.rb

```diff
@@ -19,6 +19,7 @@
       'hi.wemeditate.co' => :hi, # Hindi
       'tr.wemeditate.co' => :tr, # Turkish
       'cs.wemeditate.co' => :cs, # Czech
+      'wemeditate.sahajayoga.se' => :se, # Swedish, currently evaluating  sahajayoga.se site
```

Comment:
Could do the POC under se.wemeditate.co instead but seems closer to the end result if we create it based on the sahajayoga.se domain from the get go.

Reply:
I wrote to you on Telegram about using a wemeditate domain for the published site, but for a proof of concept use what you like. 👍